scripts/dts/python-devicetree/src/devicetree/edtlib.py

```diff
@@ -3210,16 +3227,23 @@ def _phandle_val_list(
 
     return res
 
+def _address_cells_self(node: Optional[dtlib_Node]) -> Optional[int]:
```

Comment:
nit: This could also replace or be used in `own_address_cells()`.

Reply:
Ok. Accepted. Looks better.